linux_docker_resources/Dockerfile-RHEL

```
@@ -113,6 +113,7 @@ RUN dnf install \
     libyaml-devel \
     libzstd-devel \
     $(if [ "$ROS_DISTRO" = "foxy" ]; then echo log4cxx-devel; fi) \
+    $(if test ${EL_RELEASE} != 8; then echo lttng-ust; fi) \
```

As the Debian side is getting the development files for `lttng-ust`, the RHEL side should as well.
```suggestion
    $(if test ${EL_RELEASE} != 8; then echo lttng-ust-devel; fi) \
```

The rosdep DB already reflects this properly: https://github.com/ros/rosdistro/blob/64bff7eb4ad6e18e0374219abb252a37462a94ae/rosdep/base.yaml#L4161